app/root.tsx

```diff
@@ -38,9 +58,33 @@ export default function App() {
       <body className="h-full">
         <Outlet />
         <ScrollRestoration />
+        <script
+          dangerouslySetInnerHTML={{
+            __html: `window.ENV = ${JSON.stringify(ENV)}`,
```

Comment:
This is a hacky way to pass env variables to the browser since we will need to initialize Sentry on the client as well. Open to other ideas on how to do this.